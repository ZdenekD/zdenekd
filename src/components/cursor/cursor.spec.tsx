import {render, screen} from '@testing-library/react';
import Cursor from './index';

describe('Cursor', () => {
    it('renders inner cursor without error', () => {
        render(<Cursor />);

        expect(screen.getByTestId('component-inner-cursor')).toBeInTheDocument();
    });

    it('renders outer cursor without error', () => {
        render(<Cursor />);

        expect(screen.getByTestId('component-outer-cursor')).toBeInTheDocument();
    });
});
