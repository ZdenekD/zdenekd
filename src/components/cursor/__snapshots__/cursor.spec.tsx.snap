// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cursor match snapshot 1`] = `
<i class="outerCursor"
   data-test="component-outer-cursor"
>
</i>
<i class="innerCursor "
   data-test="component-inner-cursor"
>
</i>
`;
